study-builder/studies/atcp/assent.conf

```
@@ -302,13 +302,13 @@
           "state": "INCOMPLETE",
           "height": 25,
           "width": 25,
-          "1x": "https://storage.googleapis.com/dev-at-assets/icons/Assent_itemDot_purple.svg"
+          "1x": "https://storage.googleapis.com/${assetsBucketName}/icons/Assent_itemDot_purple.svg"
```

I don't think HOCON handles substitutions inside strings. I think you need to do _string concatenation_
```suggestion
          "1x": "https://storage.googleapis.com/"${assetsBucketName}"/icons/Assent_itemDot_purple.svg"
```

---

@yufengwng's correct- it does not. From the [Concatenation](https://github.com/lightbend/config#concatenation) section of the HOCON [docs](https://github.com/lightbend/config)
```
Note: the ${} syntax must be outside the quotes!
```